interfaces/Portalicious/src/locale/messages.xlf

```diff
@@ -1580,6 +1580,21 @@
       <trans-unit id="language-switcher-selected" datatype="html">
         <source>Language: <x equiv-text="{{ selectedLanguageLabel() }}" id="INTERPOLATION"/></source>
       </trans-unit>
+      <trans-unit id="3570735198692940811" datatype="html">
+        <source>Add a reason</source>
```

Comment:
I think in other places we have just `Reason` (or something else that isn't  `Add a reason`. I would do the same everywhere to avoid adding a new translation string (and inconsistency).

Reply:
@aberonni 
We have other two occurences of a `reason` field

1. `edit-personal-information.component.html`
Label: `Write a reason for the update`
2. `change-status-reason.component.html`
Label: `Add a reason for {{ statusVerbProgressive() | lowercase }}`

Should we just change all of them to `Reason`?

Reply:
Ha, I would vote for changing them all to `Write a reason for the update`

Reply:
The only problem is with the second example, where we would need to remove `{{ statusVerbProgressive() | lowercase }}`

Reply:
I think that's a good thing. We can get rid of `REGISTRATION_STATUS_VERB_PROGRESSIVE` which is only used here, and might not even make sense for non-english languages.